lib/ts/recipe/thirdparty/index.ts

```diff
@@ -41,8 +41,16 @@ export default class Wrapper {
         return ThirdParty.getInstanceOrThrow().signOut();
     }
 
-    static async isEmailVerified(): Promise<boolean> {
-        return ThirdParty.getInstanceOrThrow().emailVerification.isEmailVerified();
+    static async isEmailVerified(userContext?: any): Promise<{
```

Comment:
Make this an optional object instead of a direct param (in other recipes too)

Reply:
Done, also refactored checking for undefined and defaulting to `{}` to use a util method instead of having the check everywhere